frontend/src/modules/otherlocales/selectors.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+
+import { createSelector } from 'reselect';
+
+import api from 'core/api';
+
+import type { LocalesState } from '.';
+import type { UserState } from 'core/user';
+
+
+const otherLocalesSelector = (state): LocalesState => state.otherlocales;
+const userSelector = (state): UserState => state.user;
```

Comment:
He he he, pretty sure that's a mistake I introduced myself again, but here we should use the `NAME` constant from each module instead of hard-coding the module names. :grin: